h2o-algos/src/main/java/hex/tree/SharedTreeModelWithContributions.java

```diff
@@ -3,6 +3,7 @@
 import hex.genmodel.algos.tree.SharedTreeNode;
 import hex.genmodel.algos.tree.SharedTreeSubgraph;
 import hex.genmodel.algos.tree.*;
+import hex.genmodel.attributes.parameters.Pair;
```

Comment:
please don't put any scoring related logic into hex.genmodel.attributes package; this belongs in algos

Reply:
Done